conditional/__init__.py

```diff
@@ -56,3 +62,7 @@ def default_route():
 def zoo():
     from conditional.models.migrate import free_the_zoo
     free_the_zoo(app.config['ZOO_DATABASE_URI'])
+
+
+app.run(host=app.config['IP'], port=app.config['PORT'])
```

Comment:
This line should remain in `app.py` with the `if __name__ == "__main__"` check to keep compatibility with other web servers.